cmd/create/cluster/cmd.go

```diff
@@ -1161,7 +1161,11 @@ func run(cmd *cobra.Command, _ []string) {
 	// OpenShift version:
 	version := args.version
 	channelGroup := args.channelGroup
-	versionList, err := versions.GetVersionList(r, channelGroup, isSTS, isHostedCP, true, true)
+	filterHostedCP := true
+	if !isHostedCP {
+		filterHostedCP = false
+	}
+	versionList, err := versions.GetVersionList(r, channelGroup, isSTS, isHostedCP, filterHostedCP, true)
```

Comment:
Isn't the same value isHostedCP == filterHostedCP ?

Reply:
I'm a little bit confused about this, do you mean using the equality of those 2 args in place of themselves in the func call?